test-scripts/tomcat-deployment/ansible/site.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Same as the above comment

Reply:
Fixed with 0e68a5bf790f62747c93c792686566a1312d6575